specification/cognitiveservices/data-plane/SpellCheck/V1.0/SpellCheck.json

```diff
@@ -0,0 +1,468 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Spell Check API - v7",
```

Comment:
(discussed with @amarzavery as well)
Please change to `Spell Check API` (you can put v7 in the description if you'd like)